quarkus-mandrel-builder-image/mandrel.yaml

```diff
@@ -34,3 +35,21 @@ images:
         arch: amd64
       - sha: 3179c87dc9336a0283a5da2c3ba74ddbd4fbec52d9d8dd78cb6979b34ce8464a
         arch: arm64
+
+  - graalvm-version: 23.0.1.2-Final
+    java-version: 17
+    tags: 23.0-java17, 23.0-jdk-17, jdk-17, jdk-17.0.7
+    variants:
+      - sha: cc73adfae094d05367e008d8ee3eebf7a08a3b45b504e55771dc739a7faabe3b
+        arch: amd64
+      - sha: 373d8341cfa678e3bdffeb8936c3edd8c353b19fdd54ba4cbe6711c6c1faa127
+        arch: arm64
+
+  - graalvm-version: 23.0.1.2-Final
+    java-version: 20
+    tags: 23.0-java20, 23.0-jdk-20, jdk-20, jdk-20.0.1
```

Comment:
This tag is also in error, 20.0.2

Reply:
Handled in https://github.com/quarkusio/quarkus-images/pull/247